data/json/monsters/zed_misc.json

```diff
@@ -89,6 +89,59 @@
     },
     "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "STUMBLES", "WARM", "POISON", "GRABS", "NO_BREATHE", "FILTHY" ]
   },
+  {
+    "id": "mon_zombullfrog",
+    "type": "MONSTER",
+    "name": { "str": "zombullfrog" },
+    "description": "The wound covered skin of this large bullfrog appears to be peeling in places, revealing a black goop beneath the surface.  Its vacant stare passes by anything that isn't food, and it will leap to devour anything it can.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "120 L",
+    "weight": "150 kg",
+    "hp": 120,
+    "speed": 50,
+    "material": [ "flesh" ],
+    "symbol": "F",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "bash", "amount": 5 } ],
```

Comment:
They could stand a bit more damage output - their specials don't hurt their target, and 23 bash armor isn't unreasonable to get by the early midgame.

Reply:
I made it 4d6+5 at melee skill 4
the frog mother is 5d6+5 at melee skill 5 (plus all the tiny tadpoles)
Might need some balancing over time.
Getting this all pushed now.